### ISSUE TITLE
Session Fixation

### ISSUE THREAD

**Issue by AnusyaAngamuthu** [robiso/wondercms#64]
Wonder CMS 2.5.1 is prone to session fixation attack.
1.The Session Fixation attack fixes a session on the victim's browser, so the attack starts before the user logs in.
2.An attacker creates a new session on a web application and records the associated session identifier. The attacker then causes the victim to authenticate against the server using the same session identifier.
3.The attacker can access to the user's account through the active session
4.When authenticating a user, it doesn’t assign a new session ID, making it possible to use an existent session ID.
Let's see the session values before login

![before login](https://user-images.githubusercontent.com/36154722/42881022-fb7c1810-8ab3-11e8-88bc-ac8e0a810115.PNG)

Session Values after Login:

![afterlogin](https://user-images.githubusercontent.com/36154722/42881072-26e29a74-8ab4-11e8-8275-4a9d8dcae3ae.PNG)

Mitigation:
The application should always first invalidate the existing session ID before authenticating a user, and if the authentication is successful, provide another sessionID.

**Comment by robiso** [comment]
Thanks @AnusyaAngamuthu, I'll look into this right away and start preparing a patch.
Any chance you could send a PoC of a successful attack to report[-at-]wondercms.com?

**Comment by AnusyaAngamuthu** [comment]
Can I send POC in Video format?

**Comment by AnusyaAngamuthu** [comment]
First Name: Anusya
Last Name: Angamuthu
Website Link : https://in.linkedin.com/in/anusya-angamuthu

**Comment by robiso** [comment]
Absolutely, the format can be video. Just to check, will you be uploading it somewhere and linking it in the email? Asking because I'm not sure if we can accept files larger than 25mb through email.

**Comment by AnusyaAngamuthu** [comment]
I will upload in youtube of my account and I will share the link with access protection.Is that fine for you?

**Comment by robiso** [comment]
Anusya that would be great!

**Comment by AnusyaAngamuthu** [comment]
Issue is fixed in version 2.5.2.Thanks for your immediate response

**Comment by robiso** [comment]
An additional thanks for confirming the fix @AnusyaAngamuthu. :+1: 

As mentioned, we've publicly thanked you on the two mentioned URLs above.

**Comment by AnusyaAngamuthu** [comment]
Thank You :)
